src/components/Modals/Receive/ReceiveInfo.tsx

```diff
@@ -123,6 +123,9 @@ export const ReceiveInfo = ({ asset, accountId }: ReceivePropsType) => {
     }
   }
 
+  const onAccountChange = (accountId: string) => {
+    setAccountId(accountId)
+  }
```

Comment:
suggestion: You could simply pass `setAccountId` down as an `onChange` prop since they have the exact same signature and one is simply calling the other, something something stack trace OCD

If you decide to keep it, wrap it in a `useCallback()` and name it `handleAccountChange`
https://jaketrent.com/post/naming-event-handlers-react